login-workflow/docs/authentication-workflow.md

```diff
@@ -2,8 +2,73 @@
 
-## AuthContextProvider
+
+## Okta Approach
+The Okta approach leverages Okta's identity and access management services to handle user authentication. This workflow includes screens related to user authentication such as Okta Redirect Login, Forgot Password, Reset Password, etc. By integrating with Okta, you can offload the complexities of authentication and focus on building your application.
+
+### OktaAuthContextProvider
+
+The screens in this workflow access shared data / configuration / API definitions through an `OktaAuthContextProvider` which should wrap all of the relevant routes / screens.
+
+You must supply the `OktaAuthContextProvider` with the following props / data:
+-   `language`: configures the language displayed on the screens
+-   `navigate`: a function that can be called to navigate to a new route
+-   `routeConfig`: an object describing the URLs you are using for the relevant routes so the workflow can correctly navigate between screens
+
+More information about the required and optional props can found in the [API](#okta-auth-context-provider) section.
+
+### Example
+
+Here is an example of how you would set up the Okta Redirect Login workflow using our recommended routing solution ([React Router](https://reactrouter.com/)).
+
+Each feature/screen from the Okta Auth Workflow that you wish to use should be rendered on a separate route.
+
+```tsx
+<Routes>
+    {/* Wrap all routes in a single shared OktaAuthContextProvider */}
+    <Route
+        element={
+            <OktaAuthContextProvider
+                language={'en'}
+                navigate={navigate}
+                routeConfig={{}}
+            >
+                <Outlet />
+            </OktaAuthContextProvider>
+        }
+    >
+        {/* Routes for each workflow screen you want to include */}
+        <Route/>
+        <Route/>
+        <Route/>
+        <Route/>
+    </Route>
+    ...
+</Routes>
+```
+
+For a detailed explanation of setting up routes, see the [Routing](./routing.md) guide.
+
+### Okta Auth Context Provider API
+
+### OktaAuthContextProviderProps
+
+| Prop Name | Type | Description | Default |
+|---|---|---|---|
+| language* | `string` | The language code specifying which language to use for the UI | `'en'` |
+| navigate* | `(url: string) => void` | A function that is used to navigate to a new URL. This is used to navigate to the various screens of the authentication workflow. |  |
+| routeConfig* | `RouteConfig` | An object that defines the various routes for the authentication workflow. See [RouteConfig](#routeconfig) for more information. |  |
+| i18n | `i18n` | An optional i18n object that is used to translate the UI. This is only needed if you want to use custom translation keys / languages inside any of the workflow screens |  |
+| errorConfig | `ErrorContextProviderProps` | An object that is used to configure error handling within the workflow. See [Error Management](./error-management.md) for more information. |  |
```

Comment:
Is it possible for us to get errors from Okta? Is this needed? Like what happens if the Okta login fails — do we get an error thrown that is captured by the Error Manager or no?